### PR TITLE
feat/update v2 contracts

### PR DESCRIPTION
**Pull request by corey-wombat**

### CODE REVIEW

File: contracts/wombat-core/test/MockStakedEth.sol
Comment (jack-wombat) [inline contracts/wombat-core/test/MockStakedEth.sol:1]: no /test/ contracts

File: contracts/wombat-core/libraries/BytesLib.sol
Comment (jack-wombat) [inline contracts/wombat-core/libraries/BytesLib.sol:1]: remove this

File: contracts/wombat-governance/interfaces/IBribe.sol [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
```
Comment (jack-wombat) [inline contracts/wombat-governance/interfaces/IBribe.sol:1]: Per https://github.com/wombat-exchange/v1-core#licensing, interface should be GPL-3.0. THis is consistent with other interfaces. Also, please update upstream afterwards.

File: contracts/wombat-governance/rewarders/MultiRewarderPerSec.sol [hunk 2]
```diff
@@ -0,0 +1,324 @@
+// SPDX-License-Identifier: GPL-3.0
```
Comment (jack-wombat) [inline contracts/wombat-governance/rewarders/MultiRewarderPerSec.sol:1]: Should be BUSDL. See https://github.com/wombat-exchange/v1-core#licensing

**Review by jack-wombat — COMMENTED** [review]
Please squash them into one commit instead of two. Otherwise, the other contracts will still be available through git commit history.

@drop19 Please also take a look. I have checked file names and license.

**Comment by drop19** [comment]
LGTM. Please squash them and make sure it can compile. Thanks!

**Review by drop19 — APPROVED** [review]

**Pull request merged**